bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/AbstractImageDelegatingModel.java

```diff
@@ -42,11 +44,18 @@
     private Resource toBeWrapped;
     private List<String> hiddenProperties;
     private Resource imageResource;
+    private Map<String, String> inheritedProperties;
 
     protected void setImageResource(@NotNull Component component, @NotNull Resource toBeWrapped, @NotNull List<String> hiddenProperties) {
         this.toBeWrapped = toBeWrapped;
         this.component = component;
-        this.hiddenProperties = new ArrayList<String>(hiddenProperties);
+        this.hiddenProperties = new ArrayList<>(hiddenProperties);
+        this.inheritedProperties = new HashMap<>();
+    }
+
+    protected void setImageResource(@NotNull Component component, @NotNull Resource toBeWrapped, @NotNull List<String> hiddenProperties, @NotNull Map<String, String> inheritedProperties) {
```

Comment:
Good opportunity to add missing javadoc to both the previous method and this.

Reply:
Addressed in 6fa9ea0